test/wait.go

```diff
@@ -268,7 +268,8 @@ func Running(name string) ConditionAccessorFn {
 		if c != nil {
 			if c.Status == corev1.ConditionTrue || c.Status == corev1.ConditionFalse {
 				return true, fmt.Errorf(`%q already finished`, name)
-			} else if c.Status == corev1.ConditionUnknown && (c.Reason == "Running" || c.Reason == "Pending") {
+			} else if c.Status == corev1.ConditionUnknown &&
+				(c.Reason == "Running" || c.Reason == "Pending" || c.Reason == "Waiting") {
```

Comment:
see above comment about reusing "Running" so that we don't have to change function which is used in other types besides "Run"

Reply:
Makes sense, done!